build/tools/Automation/checkPipelineStatus.sh

```diff
@@ -0,0 +1,40 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+# This script will check the pipeline status' result
+# for a given a pipeline invocation id.
+# A status.json file contains the meta data for a pipeline invocation id
+# of the point in time it was called, until the result has 'succeeded'.
+# The 'buildNumber' field of status.json is used later in the workflow
+# to update constants.yaml runtime image tag.
+
+pipelineInvocationId=$1
+maxRetries=3
+if [[ -n "${MAX_RETRIES}" ]]; then
+  maxRetries=${MAX_RETRIES}
+fi
+
+timeoutSeconds=10
+if [[ -n "${TIMEOUT_SECONDS}" ]]; then
+  timeoutSeconds=${TIMEOUT_SECONDS}
+fi
+
+retryCount=0
+while [ "$retryCount" -le "$maxRetries" ]
+do
+	echo "retry $retryCount"
+	# store response in status.json for later use
+	az pipelines runs show --id ${pipelineInvocationId} --organization https://devdiv.visualstudio.com/ --project DevDiv > /tmp/status.json
+	result=$( cat /tmp/status.json | jq ".result" | tr -d '"' )
+	echo "pipeline ${pipelineInvocationId} invocation result: $result"
+	if [[ "$result" == "succeeded" ]]; then
+		exit 0
```

Comment:
Nit: please consider adding a `success` echo or something. Not a big deal, but when investigating issues it feels good to see that we were successful for sure, as the exit code might not be as visible (not sure if that's the case here).

Reply:
[This line](https://github.com/microsoft/Oryx/pull/1766/files#diff-0e2923dd4411ce28feba9070d3da26c568fbe188982c47f9eddc59732c644cb5R30) `echo`s the status every iteration before an exit is performed. Which means `result` would have been printed. Is this what you were referring to?

Reply:
Yes OK, looks good